msal/__main__.py

```diff
@@ -204,7 +204,8 @@ def _main():
             header="Input authority (Note that MSA-PT apps would NOT use the /common authority)",
             accept_nonempty_string=True,
             ),
-        allow_broker=allow_broker,
+        enable_broker_on_windows=enable_broker,
```

Comment:
We used to use `allow_broker` with the meaning try enable it but ignore if it is not available.

Do we want to change the behavior to enable it or raise error if we fail to enable it?

Reply:
@bgavrilMS , fyi, @xiangyan99 also asked a follow-up question offline. "Now that we have two `enable_broker_on_windows` and `enable_broker_on_mac`, aren't they mutually exclusive? An app cannot be running on Windows and Mac *at the same time*".

I think @xiangyan99 made a good point here.

The wording "enable something" was only appropriate when the first commit of this PR used *only one* `enable_broker` parameter and was expecting app developers to set it conditionally per platform. In that case, "enable_broker=True" at runtime would indeed mean "enable it right now, or raise error if we fail to (acquire tokens via broker)".

Now we switched to the *two parameters* approach, then the meaning should really be `allow_broker_on_windows` and `allow_broker_on_mac`.